app/src/test/java/mozilla/lockbox/presenter/TestApplication.kt

```diff
@@ -0,0 +1,15 @@
+package mozilla.lockbox.presenter
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import mozilla.lockbox.LockboxApplication
+import mozilla.lockbox.log
+
+@ExperimentalCoroutinesApi
+class TestApplication : LockboxApplication() {
+
+    init {
+        log.info("Using TestApplication")
+    }
+
+    override val unitTesting = true
+}
```

Comment:
I'm not sure this is useful, given that we have `isTesting` methods.

Reply:
The application was failing on Leak Canary, even before it reached the isTesting method. I'm going to file a followup for this